noxfile.py

```diff
@@ -50,43 +60,31 @@ def coverage(session):
     session.run("coverage", "report")
 
 
+# --- Formatting ---
+
+
 @nox.session(python=CURRENT_PYTHON, tags=("format", "all"))
 def black(session):
     """Run black"""
     session.install("black")
     session.run("black", ".")
 
 
-@nox.session(python=CURRENT_PYTHON, tags=("all"))
-def docs(session):
-    """Build docs"""
-    session.install(
-        *DEPENDENCIES,
-        *OPTIONAL_DEPENDENCIES["docs"],
-        *OPTIONAL_DEPENDENCIES["azure"],
-        *OPTIONAL_DEPENDENCIES["aws"],
-    )
-
-    session.run("sphinx-build", "-b", "html", "docs", "docs/_build/html")
-
-
 @nox.session(python=CURRENT_PYTHON, tags=("format", "all"))
 def isort(session):
     """Run isort"""
     session.install("isort")
     session.run("isort", ".")
 
 
+# --- Linting ---
+
+
```

Comment:
Is there a one command to do all lintings?

Reply:
If you notice the tags argument to session, these are groups, so you can do `nox -vt lint` or `nox -vt format` and it will run anything with that tag. We can set the tags however we want.  You can preview what would run with `-l`, for example:

```console
$ nox -lt all
Nox configuration file
See https://nox.thea.codes/en/stable/config.html

Sessions defined in /srv/Development/lisa/noxfile.py:

* test -> Run tests
* example -> Run example
* coverage -> Check test coverage
* black -> Run black
* isort -> Run isort
* flake8 -> Run flake8
* mypy -> Run mypy
- docs -> Build docs
- dev -> Create virtual environment for development

sessions marked with * are selected, sessions marked with - are skipped
```

Reply:
can we create a single task to include lint, format, mypy, test, for example "check"? So if people want to submit a PR, they can run one command to check all.